Libraries/LibDebug/DebugInfo.cpp

```diff
@@ -124,6 +127,9 @@ void DebugInfo::prepare_lines()
             auto start_index = file_path.index_of(serenity_slash).value() + serenity_slash.length();
             file_path = file_path.substring(start_index, file_path.length() - start_index);
         }
+        if (file_path.starts_with("./") && !m_source_root.is_null()) {
+            file_path = LexicalPath(String::format("%s/%s", m_source_root.characters(), file_path.characters())).string();
```

Comment:
Please use `String::formatted()` in new code. You can also use `LexicalPath::canonicalized_path()`	for a slight boost in expressiveness:
```cpp
file_path = LexicalPath::canonicalized_path(String::formatted("{}/{}", m_source_root, file_path));
```